plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -191,18 +201,48 @@ public void testCreateTableIfNotExists()
         }
     }
 
-    @Test
-    public void testEmptyProjection()
+    @Test(dataProvider = "supportedTableTypesDDL")
+    public void testEmptyProjection(TableDefinition.Type tableType, String createSql, String dropSql)
     {
         // Regression test for https://github.com/trinodb/trino/issues/14981
-        try (TestTable table = new TestTable(
-                getQueryRunner()::execute,
-                "test_emtpy_projection",
-                " AS SELECT * FROM region")) {
-            assertQuery("SELECT count(*) FROM " + table.getName(), "VALUES 5");
+        String name = TEST_SCHEMA + ".test_empty_projection_" + tableType.name().toLowerCase(ENGLISH) + randomNameSuffix();
+        try {
+            onBigQuery(createSql);
+            assertQuery("SELECT count(*) FROM " + name, "VALUES 5");
+            assertQuery("SELECT count(*) FROM " + name + " WHERE regionkey = 1", "VALUES 1");
+            assertQuery("SELECT count(name) FROM " + name + " WHERE regionkey = 1", "VALUES 1");
```

Comment:
this now seems to make `testCountAggregationView` redundant. @ebyhr Can you check and if yes, I'll remove `testCountAggregationView`.